src/test/java/Homework17.java

```diff
@@ -0,0 +1,64 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+    @Test
+    public static void addSongToPlaylist() throws InterruptedException {
+
+        enterEmail("tasiakwiggins@gmail.com");
+        enterPassword("te$t$tudent");
+        submitButton();
+        Thread.sleep(2000);
+        viewAllSongs();
```

Comment:
these methods need to be placed in a separate class, take your time, we haven’t gone through this yet